### PR TITLE
Adding logging framework for services in backend. 

### PR DESCRIPTION
**Pull request by eiriksanderfjeld**

### CODE REVIEW

File: backend/api/Services/ProjectService.cs [hunk 1]
```diff
@@ -54,18 +60,26 @@ public ProjectDto CreateProject(Project project)
             project.WellProjects = new List<WellProject>();
             project.Explorations = new List<Exploration>();
 
+            Activity.Current?.AddBaggage(nameof(project), JsonConvert.SerializeObject(project, Formatting.None,
```
Comment (petterwildhagen) [inline backend/api/Services/ProjectService.cs:63]: I uncommented this and both the tests and application runs. This can be deleted?

**Review by petterwildhagen — CHANGES_REQUESTED** [review]
Delete all occurences of Activity.Current?.AddBaggage(nameof(_context.Projects), JsonConvert.SerializeObject(_context.Projects, Formatting.None,
                        new JsonSerializerSettings()
                        {
                            ReferenceLoopHandling = ReferenceLoopHandling.Ignore
                        }));

Otherwise OK. This PR seems to be about getting the framework up and running, we could do a session in the whole team to look at what we should be logging????

File: backend/api/Services/ProjectService.cs (hunk 1 above)
Comment (eiriksanderfjeld) [reply backend/api/Services/ProjectService.cs:63]: Its used to add object to request to Application Insight to enrich the logging with the actual object.

**Review by petterwildhagen — APPROVED** [review]
LGTM after clarifications with Eirik

**Pull request merged**